### PR TITLE
bump luet-cosing + cosign

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: packages/meta/collection.yaml [hunk 1]
```diff
@@ -83,7 +83,7 @@ packages:
     requires:
       - category: toolchain
         name: cosign
-        version: "<1.4"
+        version: ">=1.4"
```
Comment (mudler) [inline packages/meta/collection.yaml:86]:
```suggestion
        version: ">1.3.1"
```

Comment (mudler) [reply packages/meta/collection.yaml:86]: Maybe we should also add a comment on why? (and link to the issues.. so we don't forget in the future?)

Comment (Itxaka) [reply packages/meta/collection.yaml:86]: Trying to see first why locally validation passes but not in ci...

**Comment by mudler** [comment]
@Itxaka [master](https://github.com/rancher-sandbox/cOS-toolkit/runs/4582855173?check_suite_focus=true) seems back to failing on publishing signatures, maybe the luet-plugin in the tree is incompatible with cosign 1.3.x?

**Comment by Itxaka** [comment]
> @Itxaka master seems back to failing on publishing signatures, maybe the luet-plugin in the tree is incompatible with cosign 1.3.x?

weird, the fulcio-url is in 1.3.1 .... and the TUF_ROOT as well...

https://github.com/sigstore/cosign/commit/d401496d12bb592c5247d9414faea4985696e00c
https://github.com/sigstore/cosign/commit/bfd42e554d60b732a6403b7111ac840942e84572

**Comment by Itxaka** [comment]
force merged https://github.com/rancher-sandbox/cOS-toolkit/pull/999 to get the logs

**Comment by Itxaka** [comment]
That is still using 0.8 thougth, so it may have the same issue of parallel stuff accessing the same TUF_ROOT

**Comment by Itxaka** [comment]
Makes no sense to wait for this anymore as it wont affect anything in this pipeline once it passes the build target.

**Pull request merged**